### DISCUSSION TITLE
Can't selling or buying claims

### DISCUSSION THREAD

**Discussion by smile-minecraft** [GriefPrevention/GriefPrevention#1774] [Q&A] [answered]
Hello, when I used the /ClaimBuy command, It said "Sorry, Buying and Selling claim blocks is disabled."
I try to set the config, but I can't found any options to setup, what should I do to fix it?

**Comment by Jikoo** [comment, marked as answer]
https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#economy-integration

**Reply by smile-minecraft** [reply]
It's said "Vault was unable to find a supported economy plugin."
But I installed "Ultimate_economy" plugin,(It said it hooks into vault as economy plugin)
So should I install another economy plugin like EssentialsX?

**Reply by Jikoo** [reply]
To be clear, this message comes from GP and occurs when you use the command to buy or sell claim blocks the first time? If that's the case yes, the economy is not available through Vault, so GP cannot use it.

More info would be useful - GP version, actual logs, etc.
